packages/vc-verification/src/resolver/idata-cache.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * An Interface for Data aggreator to cache responses from Blockchain and SSI-Hub
+ */
+export interface IDataCache {
+  /**
+   * Stores role credential
+   * @param did user DID
+   * @param role namespace
+   * @param data Credential of the user for the namespace
+   */
+  setRoleCredential(did: string, role: string, data: any): void;
+
+  /**
+   * Stores DID Document
+   * @param did user DID
+   * @param data DIDDocument of the user (DID)
+   */
+  setDIDDocument(did: string, data: any): void;
```

Comment:
Similar to @JGiter 's comment about `setRoleCredential`, is it possible to use an interface for a `DIDDocument` here rather than `any`?

At a minimum, if there is no suitable interface, it is better to use `unknown` than `any`, I believe: https://stackoverflow.com/questions/51439843/unknown-vs-any